modules/configuring-quota-management-for-distributed-workloads.adoc

```diff
@@ -0,0 +1,122 @@
+:_module-type: PROCEDURE
+
+[id="configuring-quota-management-for-distributed-workloads_{context}"]
+= Configuring quota management for distributed workloads
+
+[role='_abstract']
+Configure quotas for distributed workloads on a cluster, so that you can share resources between several data science projects.
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+* You have sufficient resources. In addition to the base {productname-short} resources, you need 1.1 vCPU and 1.6 GB memory to deploy the distributed workloads infrastructure.
+* The resources are physically available in the cluster.
++
+[NOTE]
+====
+{productname-short} currently supports only a single Cluster Queue per cluster (that is, homogenous clusters), and only empty resource flavors.
+For more information about Kueue resources, see the link:https://kueue.sigs.k8s.io/docs/concepts/[Kueue documentation].
+====
+
+
+.Procedure
+ifdef::upstream,self-managed[]
+. In the {openshift-platform} console, click *Operators* -> *Installed Operators*.
+endif::[]
+ifdef::cloud-service[]
+. In the OpenShift console, click *Operators* -> *Installed Operators*.
+endif::[]
+ifdef::self-managed,cloud-service[]
+. Search for the *Red Hat OpenShift AI* Operator, and then click the Operator name to open the Operator details page.
+endif::[]
+ifdef::upstream[]
+. Search for the *Open Data Hub Operator*, and then click the Operator name to open the Operator details page.
+endif::[]
+. Click the *Data Science Cluster* tab.
+. Click the default instance name to open the instance details page.
+ifndef::upstream[]
++
+[NOTE]
+====
+Starting from {productname-long} 2.4, the default instance name for new installations is *default-dsc*.
+The default instance name for earlier installations, *rhods*, is preserved during upgrade.
+====
+endif::[]
+. Click the *YAML* tab to show the instance specifications.
+. Define an empty Kueue resource flavor, as follows:
```

Comment:
These resources are not created inside the Data Science Cluster resource but are standalone resources that would be created in openshift. This can be done by clicking on the `+` in the top right hand corner of the openshift console and pasting in the yaml. Alternatively it could be done via the oc cli by saving these files locally and using `oc apply -f filename.yaml`

Reply:
Updated to use the CLI method, thank you!